en/docs/reference/config-catalog.md

```diff
@@ -977,6 +1011,24 @@ claims_extractor_impl = "org.wso2.carbon.apimgt.impl.token.DefaultClaimsRetrieve
                             </div>
                         </div>
                     </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section>
+        <div class="mb-config-options">
+            <div class="superfences-tabs">
+
+                <input name="7" type="checkbox" id="_tab_33">
+                <label class="tab-selector" for="_tab_33"><i class="icon fa fa-code"></i></label>
+                <div class="superfences-content">
+                    <div class="mb-config-example">
+                        <pre><code class="toml">[[apim.jwt.issuer.claim_mapping]]
```

Comment:
Aren't double brackets `[[ ]]` are incorrect here?
<img width="797" alt="Screenshot 2020-10-22 at 10 51 13" src="https://user-images.githubusercontent.com/5195851/96828460-58d40e00-1455-11eb-9e5e-a9e2399e8b5d.png">

Reply:
Double brackets are added because we can have multiple such configs